### PR TITLE
Fix broken urls and change the url of Coverity Model

### LINKED ISSUE #221: broken links and 404
I have executed `make linkcheck` and I have some broken links.
```
http://docs.buildbot.net/current/manual/cfg-buildslaves.html#latent-buildslaves - Anchor 'latent-buildslaves' not found
http://news.gmane.org/ - 404 Client Error: Not Found for url: http://news.gmane.org/
http://scan5.coverity.com:8080/ - 503 Server Error: Service Unavailable for url: http://scan5.coverity.com:8080/
https://www.python.org/download/releases/3.4.4/ - 404 Client Error: OK for url: https://www.python.org/download/releases/3.4.4/
```

for news.gmane.org I think the project is dead but there is a new maintainer and the current url `news.gmane.org` does not work but there is a replacement: http://home.gmane.org/

for the release 3.4.4, I have this link: https://www.python.org/downloads/release/python-346/

for buildbot, we could remove the anchor

and scan5.coverity.com, do we continue to use it ?

thank you

### PR DESCRIPTION
**Pull request by matrixise**
Fix #221

### CODE REVIEW

File: coverity.rst [hunk 1]
```diff
@@ -150,4 +150,4 @@ Dakshesh Vyas <scan-admin@coverity.com>
 
 .. _Coverity Connect: http://scan5.coverity.com:8080/
 
-.. _coverity_model.c: https://hg.python.org/cpython/file/tip/Misc/coverity_model.c
+.. _coverity_model.c: https://raw.githubusercontent.com/python/cpython/master/Misc/coverity_model.c
```
Comment (Mariatta) [inline coverity.rst]: I think the url should be https://github.com/python/cpython/blob/master/Misc/coverity_model.c

Comment (tiran) [inline coverity.rst]: +1

**Comment by ezio-melotti** [comment]
This PR should be updated.

**Comment by willingc** [comment]
Hi @matrixise, Do you want to finish this up or would you prefer closing it? Either one is cool. Thanks.

File: buildslave.rst [hunk 2]
```diff
@@ -196,7 +196,7 @@ Latent slaves
 ^^^^^^^^^^^^^
 
 We also support running `latent buildslaves
-<http://docs.buildbot.net/current/manual/cfg-buildslaves.html#latent-buildslaves>`_
+<http://docs.buildbot.net/0.8.14/manual/cfg-buildslaves.html#latent-buildslaves>`_
```
Comment (zware) [inline buildslave.rst]: We're now on current buildbot; this page has also been updated in the meanwhile.

File: coverity.rst (hunk 1 above)
Comment (matrixise) [reply coverity.rst]: I have updated this url

**Comment by matrixise** [comment]
Hi @Mariatta and @willingc I am really so sorry, I didn't see the notification about this issue/pr. I just updated to the last version of the repo and updated the link for the coverity_model.c file.

When you have time for a review.

**Pull request merged**